test/chain-icann-lockup-test.js

```diff
@@ -571,7 +595,7 @@ describe('BIP9 - ICANN lockup (integration)', function() {
     let wdb, wallet;
 
     const FROOT = ROOT.slice();
-    const FTOP100 = TOP100.slice();
+    const FTOP100 = TOP10k.slice();
```

Comment:
Would it be better to rename occurrences of `TOP100`  to `TOP10`?  
```suggestion
    const FTOP10 = TOP10.slice();
```

At least it looks like `FTOP100` should be renamed.